python/common/org/python/types/Str.java

```diff
@@ -718,7 +718,7 @@ private long toPositiveIndex(long index) {
             __doc__ = ""
     )
     public org.python.Object isalpha() {
-        throw new org.python.exceptions.NotImplementedError("isalpha() has not been implemented.");
+        return new org.python.types.Bool(this.value.matches("[a-zA-Z]+"));
```

Comment:
Can you add a test for `print('átomo'.isalpha())` ?
This prints True in CPython, and I don't think the `[a-zA-Z]+` will match for this case.
Not sure what's the proper way of doing this, but an option could be to use [Character.isLetter()](https://docs.oracle.com/javase/7/docs/api/java/lang/Character.html#isLetter(char)).

It would be nice also to have a test for `'CasesLikeTheseWithoutSpaces'`, though it should pass with the current code.